settings/Civixero.setting.php

```diff
@@ -104,4 +110,20 @@
     ),
     'quick_form_type' => 'Element',
   ),
+  'xero_default_invoice_status' => array(
+    'group_name' => 'Xero Settings',
+    'group' => 'xero',
+    'name' => 'xero_default_invoice_status',
+    'type' => 'Array',
+    'add' => '4.4',
```

Comment:
this is misleading - I don't think there is an established alternative - perhaps remove

Reply:
You the 'add' key is misleading?

Reply:
yep - I guess so